etc/papd/print_cups.c

```diff
@@ -531,16 +505,9 @@ int cups_print_job ( char * name, char *filename, char *job, char *username, cha
 		cupsFreeDests(1,dest);
 		return (0);
 	}
-	if ((http = cupsConnectDest(dest, CUPS_DEST_FLAGS_NONE, 30000, NULL, NULL, 0, NULL, NULL)) == NULL)
-	{
-		LOG(log_error, logtype_papd,
-		    "Unable to connect to destination \"%s\": %s", dest->name, cupsLastErrorString());
-		cupsFreeDests(1,dest);
-		return (0);
-	}
 
 	info = cupsCopyDestInfo(CUPS_HTTP_DEFAULT, dest);
-
+	
```

Comment:
Nitpick: Seems unnecessary to add indentation here. ;)

Reply:
Blame Microsoft and their editor that doesn't show tab stops.